engine/runtime-language-arrow/src/test/java/org/enso/interpreter/arrow/VerifyArrowTest.java

```diff
@@ -276,7 +279,7 @@ private void populateIntArray(Value arr, byte startValue) {
   private void populateIntArray(Value arr, byte startValue, long until) {
     for (int i = 0; i < until; i++) {
       var v = startValue + i;
-      arr.setArrayElement(i, (byte) v);
+      arr.invokeMember("append", i, (byte) v);
```

Comment:
Using `append` as `arr` is no longer an Enso `Array`, good! Consider renaming `arr` to `builder`.